### PR TITLE
added structure for plotting functionality

### PR DESCRIPTION
**Pull request by kevqyzhu**
I thought that this project would allow for a large number of visualization strategies. Thus, I have organized a few of the plotting functions written by @tammy-dg and @helen-zhu  in a plotting utility function that can be deployed within the package. This should make it easier to add plotting functionality down the line as well. I am currently working on a ecdf plotting function that I will add shortly.

### CODE REVIEW

File: dnadoh/dnadoh_plot/plot_util.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+from plotly.subplots import make_subplots
+from scipy import stats
+
+# this file serves to keep all the plotting functionality in one place
+# the idea is to separate the figure creation process from the dash framework
+
+
+def get_unique_locations(df: pd.DataFrame) -> list:
+    """Get unique locations with an alternate base"""
+    return df["location"].drop_duplicates().dropna().to_list()
```
Comment (brodie-brodes) [inline dnadoh/dnadoh_plot/plot_util.py:17]: I think you can just use `df["location"].unique()` here.

Comment (brodie-brodes) [reply dnadoh/dnadoh_plot/plot_util.py:17]: Although I think yours works too!

Comment (tammy-dg) [reply dnadoh/dnadoh_plot/plot_util.py:17]: this is definitely from my convoluted code 😂 yeah I think Brodie's works better (though would want to confirm if NA's or dropped or if we still need that? 
also i think I wanted this in a list - does `df["location"].unique()`return a series or list?

**Comment by tammy-dg** [comment]
Will approve once Helen's PR is merged in!

**Comment by helen-zhu** [comment]
> Will approve once Helen's PR is merged in!

Sorry, it's in!

**Pull request merged**